ts/Maps/MapNavigation.ts

```diff
@@ -246,18 +247,45 @@ MapNavigation.prototype.update = function (
                 const unbind = addEvent(chart, 'load', (): void => {
                     // #15406: Make sure button hasnt been destroyed
                     if (button.element) {
+
+                        const expOptions = chart.options.exporting,
+                            expBtn = (
+                                expOptions &&
+                                expOptions.buttons &&
+                                expOptions.buttons.contextButton
+                            );
+
+                        // If the nav buttons are on the same side
+                        if (
+                            expBtn && defined(buttonOptions.y) &&
+                            chart.exportingGroup && chart.options &&
+                            chart.title && chart.title.textStr === '' &&
+                            chart.subtitle && chart.subtitle.textStr === '' &&
+                            (
+                                (
+                                    buttonOptions.align === 'right' &&
+                                    !expBtn.align
+                                ) ||
+                                (expBtn.align === buttonOptions.align)
+                            )
+                        ) {
+                            // (#15782) Make way for hamburger icon.
+                            buttonOptions.y +=
+                                chart.exportingGroup.getBBox().height * 1.2;
```

Comment:
Suggestion: I'm wondering whether `* 1.2` is a good approach here - it might lead to unexpected offsets depending on the button's height. What about simply adding a constant distance, e.g. 10? What do you (and others) think about it?